libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -84,27 +74,6 @@ export function toFragment(htmlStrings, ...values) {
   return fragment;
 }
 
-// TODO we might eventually want to move this to the milo core utilities
```

Comment:
The TODO is a bit misleading, the utils isn't the best place for this. I've commented below on a better alternative

Reply:
So, you are suggesting we move it to a new file, something like `libs/blocks/global-navigation/features/helpers.js` file or `libs//features/helpers.js` ?

Reply:
Since the federal feature should be used outside of the global-navigation, I'd suggest `libs/utils/federalUtils.js` (or a better name)